### ISSUE TITLE
Is there a way to set the snap indicator stroke thickness?

### ISSUE THREAD

**Issue by gamecubate** [s-leger/blender_cad_transforms#91]
The add-on's preferences don't seem to include a property for this ; would this be something set elsewhere in Blender?

**Comment by gamecubate** [comment]
The `Snap widgets line thickness` setting seems to have no effect. Would this setting have been it?

<img width="281" alt="Screen Shot 2022-09-28 at 5 33 24 AM" src="https://user-images.githubusercontent.com/1483884/192744834-fe2f39be-32df-44d1-bc72-69a364d6a527.png">

... on Blender 3.3.0 w. 0.93.2 beta 3 add-on version on macOS 10.15.7.

**Comment by s-leger** [comment]
Yes, i guess there is something wrong in blender's python gpu api.
Will check on my side soon.

**Comment by gamecubate** [comment]
Thanks!

**Comment by gamecubate** [comment]
Not sure this will help but my device (Macbook Pro) seems to be incompatible with Blender GPU rendering requirements (I am unable to set Preferences > System > Cycles Renders to Metal or any other GPU).
